students/13th/hyunheepark/westagram/user/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from user.views  import SignUpView,SignInView 
+from . import views
```

Comment:
```suggestion
from user.views  import (
    SignUpView,
    SignInView,
    IndexView
)
```

`from . import views` 를 하면 한 번에 views라는 모듈을 통째로 import 하게 됩니다.
불필요한 클래스를 사용하지 않을 수 있도록 특정 클래스를 찝어서 import 하시는 것을 추천드립니다.